akka-docs/src/main/paradox/stream/operators/StreamConverters/fromOutputStream.md

```diff
@@ -4,14 +4,34 @@ Create a sink that wraps an `OutputStream`.
 
 @ref[Additional Sink and Source converters](../index.md#additional-sink-and-source-converters)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Sink.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala) { #fromOutputStream }
-
-@@@
+@apidoc[StreamConverters.fromOutputStream](StreamConverters$) { scala="#fromOutputStream(out:()=%3Cjava.io.OutputStream,autoFlush:Boolean):akka.stream.scaladsl.Sink[akka.util.ByteString,scala.concurrent.Future[akka.stream.IOResult]]" java="#fromOutputStream(akka.japi.function.Creator)" }
 
 ## Description
 
-TODO: We would welcome help on contributing descriptions and examples, see: https://github.com/akka/akka/issues/25646
+Creates a Sink which writes incoming `ByteString`s to an `OutputStream` created by the given function.
+
+Materializes a @java[`CompletionStage`]@scala[`Future`] of `IOResult` that will be completed with the size of the file (in bytes) at the streams completion,
+and a possible exception if IO operation was not completed successfully.
```

Comment:
This is one of those operators that doesn't fail the stream on error, isn't it? That would be great to add a warning about note.

Reply:
I'm not sure. I went searching for such info in api docs and even dug in [the code](https://github.com/akka/akka/blob/6bf20f4117a8c27f8bd412228424caafe76a89eb/akka-stream/src/main/scala/akka/stream/impl/io/OutputStreamGraphStage.scala#L40-L47) and it looks like the stage is failed when the IO stream throws an exception. At least in some cases, I'm not sure the code is complete.

Reply:
That was [fixed in 2.6](https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html#iosources-fileio)